android/app/src/main/res/layout/fragment_editor_host.xml

```diff
@@ -22,17 +22,17 @@
 
       <include
         layout="@layout/toolbar_search_controls"
-        android:layout_width="match_parent"
-        android:layout_height="match_parent"
+        android:layout_width="0dp"
+        android:layout_height="?attr/actionBarSize"
         app:layout_constraintEnd_toStartOf="@id/save"
         app:layout_constraintStart_toStartOf="parent"
-        android:layout_marginEnd="?actionBarSize"
         tools:visibility="gone" />
       <TextView
         android:id="@+id/save"
         android:layout_width="wrap_content"
-        android:layout_height="match_parent"
-        android:padding="@dimen/margin_base"
+        android:layout_height="62dp"
+        android:paddingStart="12dp"
+        android:paddingEnd="12dp"
```

Comment:
```suggestion
        android:layout_height="wrap_content"
        android:padding="@dimen/margin_half_plus"
```

Your change
<img width="322" height="95" alt="after" src="https://github.com/user-attachments/assets/c04a9fd6-63d5-4cec-88ae-73f5a8a7ee03" />

Fixed
<img width="311" height="95" alt="Screenshot 2025-08-25 at 16 23 15" src="https://github.com/user-attachments/assets/9182de1e-af37-4f2c-a4fe-e4462b96bd16" />

The text was not centered vertically